mkdir: cannot create directory ‘resultsdump/test’: File exists
2024-03-24 20:26:45.607623: W external/tsl/tsl/framework/bfc_allocator.cc:296] Allocator (GPU_0_bfc) ran out of memory trying to allocate 6.09GiB with freed_by_count=0. The caller indicates that this is not a failure, but this may mean that there could be performance gains if more memory were available.
Traceback (most recent call last):
  File "/n/holyscratch01/pehlevan_lab/Lab/mletey/ICLexperiments/experiment/remote/contextlength/memory.py", line 50, in <module>
    logits = state.apply_fn({'params': state.params}, xs); # runs xs through transformer and makes predictions
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/n/holyscratch01/pehlevan_lab/Lab/mletey/ICLexperiments/experiment/remote/contextlength/../../../model/transformer.py", line 292, in __call__
    y = TransformerBlock(
        ^^^^^^^^^^^^^^^^^
  File "/n/holyscratch01/pehlevan_lab/Lab/mletey/ICLexperiments/experiment/remote/contextlength/../../../model/transformer.py", line 180, in __call__
    x = SingleHeadSelfAttention(self.config)(inputs, decoder_mask, idxs=idxs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/n/holyscratch01/pehlevan_lab/Lab/mletey/ICLexperiments/experiment/remote/contextlength/../../../model/transformer.py", line 104, in __call__
    query = dense(name='query')(inputs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/n/home05/mletey/.conda/envs/try4/lib/python3.12/site-packages/flax/linen/linear.py", line 280, in __call__
    y = dot_general(
        ^^^^^^^^^^^^
jaxlib.xla_extension.XlaRuntimeError: RESOURCE_EXHAUSTED: Out of memory while trying to allocate 6539977216 bytes.
--------------------
For simplicity, JAX has removed its internal frames from the traceback of the following exception. Set JAX_TRACEBACK_FILTERING=off to include these.
